build-parsers.sh

```diff
@@ -28,5 +28,7 @@ fi
 
 #Generate the haxe and hxml parsers
 java -jar grammar-kit.jar gen ./grammar/haxe.bnf
-java -jar grammar-kit.jar gen ./src/com/intellij/plugins/haxe/hxml/hxml.bnf
+#java -jar grammar-kit.jar gen ./src/common/com/intellij/plugins/haxe/hxml/hxml.bnf
```

Comment:
This script (I mean `./grammar/haxe.bnf`) generates code slightly different from that generated by IDEA 14 or 2016 (I tried both) with Grammar Kit 1.2.0. So I went with IDEA 14/2016 + 1.2.0. Both IDEAs produce same code, without any differences.